src/builder.ts

```diff
@@ -473,14 +473,16 @@ export class SchemaBuilder {
           typeName,
           resolve:
             field.outputType.kind === 'object'
-              ? (root, args, ctx) => {
-                  const photon = this.getPhoton(ctx)
-                  return photon[lowerFirst(mapping.model)]
-                    ['findOne']({
-                      where: { [idField.name]: root[idField.name] },
-                    })
-                    [field.name](args)
-                }
+            ? (root, args, ctx) => {
+                const photon = this.getPhoton(ctx)
+                return photon[lowerFirst(mapping.model)]
+                  ['findOne']({
+                    where: { [idField.name]: root[idField.name] },
+                  })
+                  [field.name](args)
+              }
```

Comment:
None of these lines changed, consider reversing indentation.

Reply:
Ah that's a good note! I think I had format-on-save checked in vscode and didn't notice I'd shifted everything one column to the left.